### DISCUSSION TITLE
Linux Contrib - Ubuntu Install Guide

### DISCUSSION THREAD

**Discussion by Smuzzies** [carson-katri/dream-textures#556] [Q&A]
Updated a writeup for those running linux (Ubuntu 22.04) - Install Guide

`Ubuntu 22.04 Dream Textures Installation Guide
----------------------------------------------
*******DO NOT INSTALL BLENDER VIA SNAP********
----------------------------------------------
	download blender 3.x linux version and extract to home/user/
	rename extracted folder to "blender" (/home/user/blender)
	sudo apt-get install git
	mkdir /home/user/git
	cd /home/user/git
	git clone http://github.com/carson-katri/dream-textures.git --branch releases/0.x.x
	cd dream-textures
	git submodule update --init --recursive
	cd ..
	mv dream-textures /home/user/blender/3.x/scripts/addons/
	run ./blender from /home/user/blender folder
	enable blender dev from "interface" tab
	enable dream-textures addon from "add-ons" tab
	click install dependencies button
	monitor terminal for errors
	close blender
	run ./blender from /home/user/blender folder
	download models from "add-ons" tab
----------------------------------------------
How to install NVIDIA drivers
----------------------------------------------
	sudo ubuntu-drivers  devices
	ensure you see your device listed
	sudo apt install nvidia-driver-xxx
	sudo reboot`

**Comment by THECaptGeek** [comment]
Thanks for writing this! Trying it now! Could you clarify one thing:
"download models from "add-ons" tab? 
It keeps hanging when I try it. Any suggestions?

**Comment by THECaptGeek** [comment]
@Smuzzies you're AMAZING!! I need to share your instructions!! The only extra thing I had to do (for depth map) was:
`git clone https://huggingface.co/carsonkatri/stable-diffusion-2-depth-diffusers`
But then everything worked! Now I just need to learn how to make it work "better" but at least it is functional!
THANKS!!!
